src/components/Sep6/Sep6Withdraw.tsx

```diff
@@ -225,21 +236,44 @@ export const Sep6Withdraw = () => {
     );
   }
 
-  if (sep6WithdrawAsset.status === ActionStatus.SUCCESS) {
+  if (sep6WithdrawAsset.status === ActionStatus.CAN_PROCEED) {
     return (
       <Modal visible={true} onClose={handleClose}>
-        <Heading2 className="ModalHeading">SEP-6 Withdraw Info</Heading2>
+        <Heading2 className="ModalHeading">Payment Sending</Heading2>
 
         <div className="ModalBody">
           <InfoBlock>
-            <strong>Account ID: </strong>
+            <strong>Sending Payment To: </strong>
 
-            {withdrawResponse.account_id}
+            {shortenStellarKey(withdrawResponse.account_id)}
           </InfoBlock>
 
+          <Input
```

Comment:
Making a slight adjustment to the Withdraw flow to allow user to submit payment to anchor from UI